src/plugins/marked.js

```diff
@@ -1,7 +1,7 @@
 import * as marked from 'marked'
 // import * as Showdown from 'showdown'
 // import * as showdown from 'showdown'
-import * as TurndownService from 'turndown/lib/turndown.umd.js'
+import TurndownService from 'turndown/lib/turndown.umd.js'
```

Comment:
Just curious, why UMD?

Reply:
You know that is a really good question. During the hackathon I was panicking a little bit - and was glad I got something that worked. I would have to dig into it to figure out what really the issue was, because I didn't make any git notes... :P